bridges/snowbridge/primitives/router/src/inbound/mod.rs

```diff
@@ -466,6 +466,7 @@ where
 	}
 }
 
+/// DEPRECATED in favor of [xcm_builder::ExternalConsensusLocationsConverterFor]
 pub struct EthereumLocationsConverterFor<AccountId>(PhantomData<AccountId>);
 impl<AccountId> ConvertLocation<AccountId> for EthereumLocationsConverterFor<AccountId>
```

Comment:
Can't fully deprecate, as it would the transfer. Runtime migration logic is required for:
a. migrate the owner of all erc20 assets
b. migrate the reserved account on AH for the ethereum sovereign

Reply:
`xcm_builder::ExternalConsensusLocationsConverterFor` is designed to be fully backwards-compatible with `EthereumLocationsConverterFor` so it can be replace it without breaking anything. Check the implementation and tests.

Reply:
Sorry, I didn’t check it in detail. Yeah, it’s fully backwards-compatible. Thanks for the contribution!